src/commands/serve.rs

```diff
@@ -76,6 +78,31 @@ pub(crate) fn serve(args: &ArgMatches) {
 }
 
 fn serve_foreground(args: &ArgMatches) {
+    let path = get_rymfony_project_directory().unwrap();
+    let rymfony_pid_file = path.join("rymfony.pid");
+    debug!("Looking for PID file in \"{}\".", rymfony_pid_file.to_str().unwrap());
+    if rymfony_pid_file.exists() {
+        // Check if process is rymfony and exit if true.
+
+        let infos: ServerInfo = serde_json::from_str(read_to_string(&rymfony_pid_file).unwrap().as_str()).expect("Unable to unserialize data from PID file.");
+
+        let mut system = sysinfo::System::new_all();
+        system.refresh_all();
+        let mut found = false;
+        for (pid, proc_) in system.get_processes() {
+            if pid == &infos.pid() && proc_.exe().to_str().unwrap().ends_with("rymfony") {
+                found = true;
+                break;
+            }
+        }
+
+        if found {
+            info!("The server is already running and listening to {}://127.0.0.1:{}", infos.scheme(), infos.port());
+            return;
+        }
```

Comment:
Early return can be put directly in the `for` loop

```suggestion
        for (pid, proc_) in system.get_processes() {
            if pid == &infos.pid() && proc_.exe().to_str().unwrap().ends_with("rymfony") {
                info!("The server is already running and listening to {}://127.0.0.1:{}", infos.scheme(), infos.port());
                return;
            }
        }
```